app/src/main/kotlin/erikjhordanrey/android_kotlin_devises/view/CurrencyViewModel.kt

```diff
@@ -40,6 +40,16 @@ class CurrencyViewModel(private val currencyRepository: CurrencyRepository) : Vi
     private var liveCurrencyData: LiveData<List<Currency>>? = null
     private var liveAvailableExchange: LiveData<AvailableExchange>? = null
 
+    val isProgressVisible = MutableLiveData<Boolean>().apply { value = false }
```

Comment:
you should expose as `livedata` in order to avoid altering the state in the fragment